processor/configuration/src/test/java/com/redhat/service/smartevents/processor/GatewayConfiguratorImplTest.java

```diff
@@ -35,15 +32,15 @@
 class GatewayConfiguratorImplTest {
 
     private static final Map<String, ExpectedBeanClasses<Action>> EXPECTED_ACTION_BEANS = Map.of(
-            KafkaTopicAction.TYPE, expect(KafkaTopicActionValidator.class, null),
-            SendToBridgeAction.TYPE, expect(SendToBridgeActionValidator.class, SendToBridgeActionResolver.class),
-            SlackAction.TYPE, expect(SlackActionValidator.class, SlackActionResolver.class),
+            KafkaTopicAction.TYPE, expect(DefaultGatewayValidator.class, null),
+            SendToBridgeAction.TYPE, expect(DefaultGatewayValidator.class, SendToBridgeActionResolver.class),
+            SlackAction.TYPE, expect(DefaultGatewayValidator.class, SlackActionResolver.class),
             WebhookAction.TYPE, expect(WebhookActionValidator.class, null));
 
     private static final Map<String, ExpectedBeanClasses<Source>> EXPECTED_SOURCE_BEANS = Map.of(
-            AwsS3Source.TYPE, expect(AwsS3SourceValidator.class, SourceResolver.class),
-            AwsSqsSource.TYPE, expect(AwsSqsSourceValidator.class, SourceResolver.class),
-            SlackSource.TYPE, expect(SlackSourceValidator.class, SourceResolver.class));
+            AwsS3Source.TYPE, expect(DefaultGatewayValidator.class, SourceConnectorResolver.class),
```

Comment:
I'm not sure why we need this test anymore. Is this testing that AwsSqsSource has a custom validator? In that case I think it's already tested by [AwsSqsSourceValidatorTest.java](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/830/files#diff-aecdc7f0cbdaba30d4cbcf04b6bfb3c1f0dd790ef9aba7795d7c4dd5ead41802)

Reply:
This is checking that the GatewayConfiguration actually returns the custom validator and not the default one